src/WCSimPMTObject.cc

```diff
@@ -1513,13 +1525,16 @@ G4float* BoxandLine12inchHQE::GetQEWavelength(){
 }
 
 G4float* BoxandLine12inchHQE::GetQE(){
+  G4float correctionFactor = 1./0.73;//Correction factor added in July 2015 to set R3600 C.E. to 73% and not to change the R3600 output.
```

Comment:
Remove this comment on line 1528. It doesn't apply now that we aren't changing R3600 to 73%. Replace it with the comment on line 1354 //Correction factor added in July 2015 to scale the output of B&L PDs to 1.9 times the 20" PMTS based on Hamamatsu measurement. This was done in Pull Request #98 and will be removed once a more permanent solution is found.